components/ringcentral/sources/new-inbound-message/new-inbound-message.mjs

```diff
@@ -36,18 +36,20 @@ export default {
         messageType: this.messageType,
       };
     },
-    generateMeta(data) {
-      const {
-        timestamp,
-        uuid: id,
-      } = data;
-      const summary = "New inbound message event";
-      const ts = Date.parse(timestamp);
-      return {
-        id,
-        summary,
-        ts,
-      };
+    async emitEvent(event) {
+      const { body } = event
+
+      const message = await this.ringcentral.getMessage({
+        $,
+        extensionId: this.extensionId,
+        messageId: body.uuid,
```

Comment:
The `messageId` should be done for each `body.changes[0].newMessageIds[].`

<img width="424" alt="image" src="https://user-images.githubusercontent.com/35234925/205097581-0acaee05-c2a9-4983-af0d-08bc73a0c056.png">

Reply:
Hello, I hope are all fine!
I forget about multiple messages, I will fix to emit one event for each message!